tests/test_model_metrics.cc

```diff
@@ -41,4 +41,20 @@ TYPED_TEST(ModelMetricTester, test_sanity) {
   EXPECT_FALSE(std::isnan(metric));
 }
 
-} // namespace albatross
+TEST(test_gp_from_prediction, test_model_from_prediction) {
```

Comment:
Is there an existing file this should go in or maybe a new one? Doesn't seem to belong here...

Reply:
`test_gp.cc` would probably be a good spot for it.

Reply:
note: `test_gp.cc` doesn't exist yet ... but I'm about to add it in another PR.  Feel free to create the file and add your test, or wait for my PR.

Reply:
I can wait for your PR. :)

Reply:
Actually ... I lied, my tests ended up in `test_sparse_gp.cc` which probably doesn't make sense for these.  Want to either make a new test file (`test_gp.cc`) or throw these in `test_models.cc`?